src/app/fdctl/run/tiles/fd_verify.h

```diff
@@ -38,9 +39,9 @@ typedef struct {
   ulong       out_chunk0;
   ulong       out_wmark;
   ulong       out_chunk;
-} fd_verify_ctx_t;
 
-#define FD_VERIFY_DEDUP_TAG_FROM_PAYLOAD_SIG(payload_sig_p) FD_LOAD( ulong, (payload_sig_p) )
+  fd_hashmap_t hashmap;
```

Comment:
I don't really like calling this `fd_hashmap_t` which implies it's a container for the data (the actual map itself), rather than just a hashing algorithm. Maybe just use `fd_hasher_t` ?

Reply:
it's the state of the hash function, exactly like fd_sha?_t